api/src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClient.java

```diff
@@ -34,7 +34,8 @@ void createRuntime(Runtime runtime, String workspaceNamespace, String workspaceF
   void updateRuntime(Runtime runtime) throws WorkbenchException;
 
   /** Deletes a notebook runtime */
-  void deleteRuntime(String googleProject, String runtimeName) throws WorkbenchException;
+  void deleteRuntime(String googleProject, String runtimeName, Boolean deleteDisk)
```

Comment:
nit: would prefer that this takes a primitive boolean instead so that the Java interface is as simple as possible. Looks like the calling code already forces it to be non-null, which is great.

Reply:
^ FYI: you may have missed this comment